internal/pkg/containers/containerd/containerd.go

```diff
@@ -0,0 +1,328 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
this actually got moved almost without changes, but `git` doesn't want to show that